src/shared/csv.ts

```diff
@@ -0,0 +1,13 @@
+/**
+ * Escapes provided value so that it can be used inside a CSV without breaking
+ * its format.
+ * It checks if the value contains a comma, newline or double quote, then wraps
+ * it in double quotes.
+ * It also escapes existing double quotes.
+ */
+export function escapeCSVValue(value: string): string {
```

Comment:
This can be shortened to something like "Escape a CSV field value", along with a short example and any spec this follows (eg. https://www.ietf.org/rfc/rfc4180.txt)

The code and tests provide additional details if the reader needs them.